### PR TITLE
feat(docs): document `alerta` package

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 1]
```diff
@@ -1,25 +1,71 @@
...
+// - text: - string - Alerta text description. Default is `""`.
+// - tags: - array of strings - List of event tags. Default is `[]`.
+// - attributes: - record - (Required) Alert attributes.
+// - origin: - string - monitoring component.
+// - type: - string - Event type. Default is `""`.
+// - timestamp: - time - time alert was generated. Default is `now()`.
+// 
+// ## Examples
+// ### Send the last reported value and status to Alerta
+// ```
```
Comment (nathanielc) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:33]: can you add a `no_run` to this example as we don't want it to try to connect to alerta?

**Review by nathanielc — APPROVED** [review]

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 2]
```diff
@@ -1,25 +1,71 @@
...
+//   group: "example-group",
+//   value: string(v: lastReported._value),
+//   text: "Service is ${severity}. The last reported value was ${string(v: lastReported._value)}.",
+//   tags: ["ex1", "ex2"],
+//   attributes: {},
+//   origin: "InfluxDB",
+//   type: "exampleAlertType",
+//   timestamp: now(),
+// )
+// ```
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:68]:
~~~suggestion
// ```
//
// tags: single notification
//
~~~

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 3]
```diff
@@ -59,14 +105,76 @@ alert = (
...
+//       service: r.service,
+//       group: "example-group",
+//       value: r.status,
+//       text: "Status is critical.",
+//       tags: ["ex1", "ex2"],
+//       attributes: {},
+//       type: "exampleAlertType",
+//       timestamp: now(),
+//     }
+//   })()
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:174]: `flux fmt`'d

```suggestion
// import "contrib/bonitoo-io/alerta"
// import "influxdata/influxdb/secrets"
// 
// apiKey = secrets.get(key: "ALERTA_API_KEY")
// endpoint = alerta.endpoint(
//     url: "https://alerta.io:8080/alert",
//     apiKey: apiKey,
//     environment: "Production",
//     origin: "InfluxDB",
// )
// 
// crit_events = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")
// 
// crit_events
//     |> endpoint(
//         mapFn: (r) => {
//             return {r with
//                 resource: "example-resource",
//                 event: "example-event",
//                 severity: "critical",
//                 service: r.service,
//                 group: "example-group",
//                 value: r.status,
//                 text: "Status is critical.",
//                 tags: ["ex1", "ex2"],
//                 attributes: {},
//                 type: "exampleAlertType",
//                 timestamp: now(),
//             }
//         },
//     )()
```

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 4]
```diff
@@ -59,14 +105,76 @@ alert = (
...
+// ```
+// 
+// tags: notification-endpoints
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:177]:
```suggestion
// tags: notification endpoints,transformations
//
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 5]
```diff
@@ -1,25 +1,74 @@
...
+// - resource: (Required) Resource associated with the alert.
+// - event: (Required) Event name.
+// - environment: Alerta environment. Valid values: "Production", "Development" or empty string (default).
+// - severity: (Required) Event severity. See [Alerta severities](https://docs.alerta.io/en/latest/api/alert.html#alert-severities).
+// - service: List of affected services. Default is `[]`.
+// - group: Alerta event group. Default is `""`.
+// - value: Event value.  Default is `""`.
+// - text: Alerta text description. Default is `""`.
+// - tags: List of event tags. Default is `[]`.
+// - attributes: (Required) Alert attributes.
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:26]:
```suggestion
// - url: Alerta URL.
// - apiKey: Alerta API key.
// - resource: Resource associated with the alert.
// - event: Event name.
// - environment: Alerta environment.
// 
//   **Valid values**:
//   - `"Production"`
//   - `"Development"`
//   - empty string (`""`) _(default)_
// 
// - severity: Event severity. See [Alerta severities](https://docs.alerta.io/en/latest/api/alert.html#alert-severities).
// - service: List of affected services. Default is `[]`.
// - group: Alerta event group. Default is `""`.
// - value: Event value.  Default is `""`.
// - text: Alerta text description. Default is `""`.
// - tags: List of event tags. Default is `[]`.
// - attributes: Alert attributes.
```

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 6]
```diff
@@ -1,25 +1,74 @@
...
+// - origin: monitoring component.
+// - type: Event type. Default is `""`.
+// - timestamp: time alert was generated. Default is `now()`.
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:29]:
```suggestion
// - origin: Monitoring component.
// - type: Event type. Default is `""`.
// - timestamp: Time alert was generated. Default is `now()`.
```

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 7]
```diff
@@ -1,25 +1,74 @@
...
+//   service: ["example-service"],
+//   group: "example-group",
+//   value: string(v: lastReported._value),
+//   text: "Service is ${severity}. The last reported value was ${string(v: lastReported._value)}.",
+//   tags: ["ex1", "ex2"],
+//   attributes: {},
+//   origin: "InfluxDB",
+//   type: "exampleAlertType",
+//   timestamp: now(),
+// )
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:67]: `flux fmt`'d

```suggestion
// import "contrib/bonitoo-io/alerta"
// import "influxdata/influxdb/secrets"
// 
// apiKey = secrets.get(key: "ALERTA_API_KEY")
// 
// lastReported = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(
//         fn: (r) => r._measurement == "example-measurement" and r._field == "level",
//     )
//     |> last()
//     |> findRecord(fn: (key) => true, idx: 0)
// 
// severity = if lastReported._value > 50 then "warning" else "ok"
// 
// alerta.alert(
//     url: "https://alerta.io:8080/alert",
//     apiKey: apiKey,
//     resource: "example-resource",
//     event: "Example event",
//     environment: "Production",
//     severity: severity,
//     service: ["example-service"],
//     group: "example-group",
//     value: string(v: lastReported._value),
//     text: "Service is ${severity}. The last reported value was ${string(v: lastReported._value)}.",
//     tags: ["ex1", "ex2"],
//     attributes: {},
//     origin: "InfluxDB",
//     type: "exampleAlertType",
//     timestamp: now(),
// )
```

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 8]
```diff
@@ -59,14 +108,79 @@ alert = (
...
-// `tags`, `attributes`, `origin`, `type` and `timestamp` fields as defined in the `alert` function arguments.
+// endpoint sends alerts to Alerta using data from input rows.
+//
+// ## Parameters
+// 
+// - url: (Required) Alerta URL.
+// - apiKey: (Required) Alerta API key.
+// - environment: Alert environment. Default is `""`.
+//   Valid values: "Production", "Development" or empty string (default).
+// - origin: Alert origin. Default is `"InfluxDB"`.
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:119]:
```suggestion
// - url: Alerta URL.
// - apiKey: Alerta API key.
// - environment: Alert environment. Default is `""`.
//
//   **Valid values**:
//   - `"Production"`
//   - `"Development"`
//   - empty string (`""`) _(default)_.
//
// - origin: Alert origin. Default is `"InfluxDB"`.
```

File: stdlib/contrib/bonitoo-io/alerta/alerta.flux [hunk 9]
```diff
@@ -59,14 +108,79 @@ alert = (
...
+//
+// ## Usage
+// `alerta.endpoint` is a factory function that outputs another function.
+//     The output function requires a `mapFn` parameter.
```
Comment (sanderson) [inline stdlib/contrib/bonitoo-io/alerta/alerta.flux:123]:
```suggestion
// `alerta.endpoint` is a factory function that outputs another function.
// The output function requires a `mapFn` parameter.
```

**Review by sanderson — CHANGES_REQUESTED** [review]

**Review by sanderson — APPROVED** [review]

**Pull request merged**